src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
Nah you just require like so: `import fetch from 'unfetch'` which imports [this file](https://github.com/developit/unfetch/blob/master/src/index.js). The [polyfill file](https://github.com/developit/unfetch/blob/master/polyfill.js) literally just imports that ponyfill and assigns it to `window` if not already there.